enum VerMajor
{
    VER_MAJOR = 0;
}

enum VerMinor
{
    VER_MINOR = 9;
}

enum VerBuild
{
    VER_BUILD = 1;
}
